backend/api/v1/audit.go

```diff
@@ -289,6 +289,24 @@ func getResponseString(response any) (string, error) {
 	return string(b), nil
 }
 
+func redactSetIamPolicyRequest(r *v1pb.SetIamPolicyRequest) *v1pb.SetIamPolicyRequest {
```

Comment:
What's redacted or should be redacted? If not, let's use the original request.

Reply:
binding_deltas are not needed in the request.